libs/wingsdk/src/cloud/api.ts

```diff
@@ -46,6 +46,12 @@ export interface ApiCorsOptions {
    * @default - false
    */
   readonly allowCredentials?: boolean;
+
+  /**
+   * Number of seconds that the browser should cache preflight request results.
+   * @default - 300
+   */
+  readonly maxAge?: number;
```

Comment:
This would be a good use case for a `duration`, no?
```suggestion
  /**
   * Duration that the browser should cache preflight request results.
   * @default - 300 seconds
   */
  readonly maxAge?: Duration;
```